### PR TITLE
USWDS - Documentation: Create initial security page.

### PR DESCRIPTION
**Pull request by mejiaj**
## Description

Part of https://github.com/uswds/uswds/issues/3569

1st draft of Security page.

### Preview link
[Security Information](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.app.cloud.gov/preview/uswds/uswds-site/jm-add-security-page/about/security-information/)

Before you hit Submit, make sure you’ve done whichever of these applies to you:

- [x] Follow the [18F Front End Coding Style Guide](https://pages.18f.gov/frontend/) and [Accessibility Guide](https://pages.18f.gov/accessibility/checklist/).
- [ ] Run `npm test` and make sure the tests for the files you have changed have passed.
- [ ] Run your code through [HTML_CodeSniffer](http://squizlabs.github.io/HTML_CodeSniffer/) and make sure it’s error free.
- [x] Title your pull request using this format: [Website] - [UI component]: Brief statement describing what this pull request solves.

### CODE REVIEW

File: pages/about/security-information.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+permalink: /about/security-information/
```
Comment (afeijoo) [inline pages/about/security-information.md:2]: Update to /about/security here too.

**Review by afeijoo — DISMISSED** [review]
See inline changes and comments.

File: pages/about/security-information.md (hunk 1 above)
Comment (mejiaj) [reply pages/about/security-information.md]: I changed the page to name to just `security` and updated redirects.

**Comment by mejiaj** [comment]
Ready for review. Changed the file to `Security` and updated permalinks.

**Comment by thisisdano** [comment]
This adds the page. Subsequent changes can happen either in SECURITY.md (in uswds) or in subsequent PRs.

**Pull request merged**